src/resource/page/ResourcePage.scss

```diff
@@ -35,10 +35,10 @@
   white-space: pre-wrap;
 }
 
-.child-resource-description-text {
+.child-and-parent-resource-description-text {
```

Comment:
Not sure if Im satisfied by this class name 🤔 . What do you think? It's a class for both child and parent resource description. Would it be better if it was: 'child-or-parent-resource-description-text'? One option would be just 'resource-description-text'. But I feel it's not enough explicit in what it means. Because the whole page is about the resource itself, and it includes the child and parent resources

Reply:
- [ ] Rename to related-resource-